docs/pages/pointcloud_datasets.rst

```diff
@@ -0,0 +1,37 @@
+Working with Point Cloud Datasets
+====================================
+
+.. Note:: Before using this quick guide, it will help your understanding to do the :doc:`Basic Usage Tutorial </pages/basic_usage_tutorial>`.
+
+Dataset type
+~~~~~~~~~~~~
+Point Cloud datasets are currently implemented as :doc:`Point Cloud Datasets V1 </pages/development/pointcloud_v1>`. Kart version controls point cloud tiles themselves as LAZ files, which means the tiles are checked out into the working copy without any conversion step - the contents of the working copy is an accurate reflection of the contents of the dataset itself at a particular point in time.
+
+Importing point cloud datasets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- ``kart import <tile> [<tile>] [<tile>]``
+- ``kart import --dataset=<dataset_name> <tile> [<tile>] [<tile>]``
+
+This command imports one or more point-cloud tiles into the kart repository in the
+working directory. All tiles are imported into the same dataset.
+
+The tiles to import must be LAS files of some variant - compressed tiles (LAZ) and cloud-optized tiles (COPC) are allowed.
+By default, all tiles are converted to COPC (`Cloud Optimized Point Cloud <copc_>`_) as they are imported.
+You can specify ``--keep-existing-format`` to keep them as they are.
```

Comment:
I'd prefer to steer users towards COPC rather than be totally agnostic. Is it worth adding a note about why it might be best to allow Kart to convert them? Just something simple like

> COPC is recommended as it improves the efficiency with which apps and cloud services can access large point cloud datasets.

Reply:
Done